src/tests/users-service.test.js

```diff
@@ -2,8 +2,7 @@ import {
   createUser,
```

Comment:
Why are the tests from users-service being deleted?

Reply:
They were just moved into follows-service.test.js. The follow method was originally a part of the users-service, but it was refactored to have its own service and thus test file.